src/bugsnag.js

```diff
@@ -306,6 +306,11 @@
       });
     }
 
+    // make an empty object onto which we attach our fake logging functions
+    if (typeof window.console === "undefined") {
+      window.console = {};
```

Comment:
We tend to be pretty conservative about adding things to the global scope, and I'm pretty worried about doing this, especially with something that doesn't work like a polyfill — if someone out there is doing something silly like using the presence/absence of `window.console` to do some kind of feature detection, I don't want to mess it up. It might be worth having a larger discussion to figure out what the most sensible thing to do here is, in the absence of `console.log` support.